docs/installation.rst

```diff
@@ -0,0 +1,51 @@
+.. highlight:: shell
```

Comment:
This is the standard installation doc but you may want to include something about CUDA and such.

Reply:
cuda gets automagically installed with Torch.